language/fr/profile.lang

```diff
@@ -1,5 +1,58 @@
 <?php
 
-include("{$_EnginePath}language/en/profile.lang");
+$_Lang['PageTitle']              = 'Profil de joueur';
+
+$_Lang['Table_NoAvatar']         = 'Aucune image';
+$_Lang['Table_Username']         = 'Nom d\'utilisateur';
+$_Lang['Table_Gamerank']         = 'Rang';
+$_Lang['Table_Allyname']         = 'Alliance';
+$_Lang['Table_Allyrank']         = 'Classement de l\'alliance';
+$_Lang['Table_MotherPlanet']     = 'Planète mère';
+$_Lang['Table_WritePM']          = 'Écrire un message à ce joueur';
+$_Lang['Table_Add2Buddy']        = 'Ajouter ce joueur à tes amis';
+$_Lang['Table_AllyInvite']       = 'Inviter ce joueur dans ton alliance';
+$_Lang['Table_Buddy']            = 'Ami / ';
+$_Lang['Table_Online']           = 'Connecté';
+$_Lang['Table_Offline']          = 'Non connecté';
+$_Lang['Table_Ignore']           = 'Ignorer les messages de ce joueur';
+$_Lang['Table_UnIgnore']         = 'Ne plus ignorer les messages de ce joueur';
+$_Lang['Table_Report']           = 'Signaler ce joueur';
+$_Lang['Table_UserOnVacations']  = 'Joueur en vacances';
+$_Lang['Table_UserIsBanned']     = 'Compte sanctionné par une suspension';
+$_Lang['Table_AdminInfo']        = 'Informations de l\'administrateur';
+
+$_Lang['Table_WarStats']         = 'Statistiques de combat';
+$_Lang['Table_FightsTotal']      = 'Combats';
+$_Lang['Table_FightsWon']        = 'Victoires';
+$_Lang['Table_FightsWonACS']     = '&raquo; Victoires groupées';
+$_Lang['Table_FightsDraw']       = 'Matchs nuls';
+$_Lang['Table_FightsLost']       = 'Défaites';
+$_Lang['Table_FightsInAlly']     = 'Attaques groupées';
```

Comment:
Obviously, I'm no expert in the French language, but an online translator suggests that this means "group attacks", which to me suggests union attacks (ACS attacks). The original translation ("Allied battles"), which admittedly may not be super clear as well, was meant to be a label for all fights that happened only between players within the same alliance (so for example, when member X of alliance A attacks member Y of same alliance A just to create a new moon on member Y planet's orbit).

So, do you think this needs to be changed here, or should it stay the same as it is right now?

Reply:
OK, I misunderstood the meaning then. 

To make sure: "Table_FightsInAlly" translated in English "Allied battles" means "Fights against members of the same alliance". Is that right?

Reply:
Yes, that's right.

Reply:
Ok. I have fixed that (and the corresponding translation in overview.lang too).